components/locid/src/macros.rs

```diff
@@ -191,7 +191,8 @@ macro_rules! langid {
 /// assert_eq!(DE_AT, de_at);
 /// ```
 ///
-/// *Note*: The macro cannot produce locales with more than one variant or extensions due to const
+/// *Note*: The macro cannot produce locales with more than one variant or multiple extensions
+/// (only single keyword unicode extension is supported) due to const
 /// limitations (see [`Heap Allocations in Constants`]):
 ///
 /// ```compile_fail
```

Comment:
Nit: This is a bad example, because it makes "ja" look like it is a value for "ca", but it is actually another key. Please fix the example to have a realistic locale that we don't support, such as `"sl-IT-rozaj-biske-1994"`

Reply:
Fixed.